include/rocksdb/utilities/optimistic_transaction_db.h

```diff
@@ -31,6 +31,28 @@ struct OptimisticTransactionOptions {
   const Comparator* cmp = BytewiseComparator();
 };
 
+enum class OccValidationPolicy {
+  // Validate serially at commit stage, AFTER entering the write-group.
+  // Isolation validation is processed single-threaded(since in the
+  // write-group).
+  // May suffer from high mutex contention, as per this link:
+  // https://github.com/facebook/rocksdb/issues/4402
+  VALIDATE_SERIAL = 0,
+  // Validate parallelly before commit stage, BEFORE entering the write-group to
+  // reduce mutex contention.
+  // Each txn acquires locks for its write-set records in some well-defined
+  // order.
+  VALIDATE_PARALLEL
+};
+
+struct OptimisticTransactionDBOptions {
+  OccValidationPolicy validate_policy = OccValidationPolicy::VALIDATE_PARALLEL;
```

Comment:
Default behavior change needs to be explained in HISTORY.md.

Reply:
resolved.